catalog/app/components/SearchResults/SearchResults.js

```diff
@@ -85,12 +88,12 @@ function HeaderIcon(props) {
   )
 }
 
-function ObjectHeader({ handle, showBucket, downloadable }) {
+function ObjectHeader({ handle, showBucket, downloadable = false }) {
   return (
     <Heading display="flex" alignItems="center" mb={1}>
```

Comment:
I think, you need `mb={1}` outside of `ObjectHeader`, and use `mb={0}` for directory without metadata.